lib/omniauth/publik/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Omniauth
   module Publik
-    VERSION = "0.1.0"
+    VERSION = "0.0.9"
```

Comment:
We should bump the current version to `0.1.1`

Reply:
I totally agree!

Reply:
I do it right now 😁